spec/intl.html

```diff
@@ -1338,6 +1338,259 @@ <h1>CalendarEraYear ( _calendar_, _dateLike_ )</h1>
+          </dl>
+        </emu-clause>
+
+        <emu-clause id="sec-temporal-calendardateerayear" type="abstract operation">
+          <h1>
+            CalendarDateEraYear (
+              _calendar_: a String,
+              _date_: a %Temporal.PlainDate%,
+            )
+          </h1>
+          <dl class="header">
+            <dt>description</dt>
+            <dd>It performs implementation-defined processing to find the year of the current era for the date corresponding to _date_ in the context of the calendar represented by the string _calendar_ and returns it.</dd>
+          </dl>
+        </emu-clause>
+
+        <emu-clause id="sec-temporal-calendardateyear" type="abstract operation">
+          <h1>
+            CalendarDateYear (
+              _calendar_: a String,
+              _date_: a %Temporal.PlainDate%,
+            )
+          </h1>
+          <dl class="header">
+            <dt>description</dt>
+            <dd>It performs implementation-defined processing to find the year for the date corresponding to _date_ in the context of the calendar represented by the string _calendar_ and returns it.</dd>
+          </dl>
+        </emu-clause>
+
+        <emu-clause id="sec-temporal-calendardatemonth" type="abstract operation">
+          <h1>
+            CalendarDateMonth (
+              _calendar_: a String,
+              _date_: a %Temporal.PlainDate%,
+            )
+          </h1>
+          <dl class="header">
+            <dt>description</dt>
+            <dd>It performs implementation-defined processing to find the month for the date corresponding to _date_ in the context of the calendar represented by the string _calendar_ and returns it.</dd>
+          </dl>
+        </emu-clause>
+
+        <emu-clause id="sec-temporal-calendardatemonthcode" type="abstract operation">
+          <h1>
+            CalendarDateMonthCode (
+              _calendar_: a String,
+              _date_: a %Temporal.PlainDate%,
+            )
+          </h1>
+          <dl class="header">
+            <dt>description</dt>
+            <dd>It performs implementation-defined processing to find the monthCode corresponding to the month for the date corresponding to _date_ in the context of the calendar represented by the string _calendar_ and returns it.</dd>
+          </dl>
+        </emu-clause>
+
+        <emu-clause id="sec-temporal-calendardateday" type="abstract operation">
+          <h1>
+            CalendarDateDay (
+              _calendar_: a String,
+              _date_: a %Temporal.PlainDate%,
+            )
+          </h1>
+          <dl class="header">
+            <dt>description</dt>
+            <dd>It performs implementation-defined processing to find the day for the date corresponding to _date_ in the context of the calendar represented by the string _calendar_ and returns it.</dd>
```

Comment:
Should we note that the return value is expected to be a 1-based integer? Like this:

```suggestion
            <dd>It performs implementation-defined processing to find the 1-based day of the month for the date corresponding to _date_ in the context of the calendar represented by the string _calendar_ and returns it.</dd>
```